cypress/integration/26-register_registration/creator_sees_registration_saved_with_correct_values/creator_sees_registration_saved_with_correct_values.js

```diff
@@ -1,5 +1,12 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
```

Comment:
Hvorfor importeres `And` nå når det ble brukt før også? `Given` og `Then` brukes også, uten at de importeres 🤔

Reply:
Tro det har skjedd automatisk, fjerner det om mulig